src/popup/pages/TipsClaim.vue

```diff
@@ -70,83 +71,88 @@ export default {
-      if (tab && validateTipUrl(tab.url)) {
-        this.url = tab.url;
-      }
-    }
-  },
-  methods: {
-    async claimTips() {
-      const { openModal, openDefaultModal } = useModals();
-      const { activeAccount } = useAccounts({ store: this.$store });
-
-      const url = this.normalizedUrl;
-      this.loading = true;
-      await watchUntilTruthy(() => this.sdk && this.tippingV1);
+    const tipUrl = ref('');
+    const loading = ref(false);
+    const tippingSupported = computed(() => root.$store.getters.tippingSupported);
+
+    const normalizedUrl = computed(() => (
+      validateTipUrl(tipUrl.value)) ? toURL(tipUrl.value).toString() : '');
+
+    async function claimTips() {
+      const url = normalizedUrl.value;
+      loading.value = true;
       try {
+        const { tippingV1 } = await getTippingContracts();
         const claimAmount = parseFloat(
           aettosToAe(
-            await this.tippingV1.methods
-              .unclaimed_for_url(url)
+            await tippingV1.value.unclaimed_for_url(url)
               .then((r) => r.decodedResult)
               .catch(() => 1),
           ),
         );
         if (!claimAmount) {
           throw new Error('NO_ZERO_AMOUNT_PAYOUT');
         }
-        await this.$store.dispatch('claimTips', { url, address: activeAccount.value.address });
-        await this.$store.dispatch('cacheInvalidateOracle');
-        await this.$store.dispatch('cacheInvalidateTips');
+        await root.$store.dispatch('claimTips', { url, address: activeAccount.value.address });
+        await Promise.all([
+          root.$store.dispatch('cacheInvalidateOracle'),
+          root.$store.dispatch('cacheInvalidateTips'),
+        ]);
 
         openModal(MODAL_CLAIM_SUCCESS, { url, claimAmount });
 
-        this.$router.push({ name: ROUTE_ACCOUNT });
-      } catch (e) {
+        root.$router.push({ name: ROUTE_ACCOUNT });
+      } catch (e: any) {
         const { error = '' } = e.response ? e.response.data : {};
         let msg;
-        if (error.includes('MORE_ORACLES_NEEDED')) msg = this.$t('pages.claim.moreOracles');
-        else if (error.includes('URL_NOT_EXISTING')) msg = this.$t('pages.claim.urlNotExisting');
-        else if (
+        if (error.includes('MORE_ORACLES_NEEDED')) {
+          msg = root.$t('pages.claim.moreOracles');
+        } else if (error.includes('URL_NOT_EXISTING')) {
+          msg = root.$t('pages.claim.urlNotExisting');
+        } else if (
           error.includes('NO_ZERO_AMOUNT_PAYOUT')
           || e.message.includes('NO_ZERO_AMOUNT_PAYOUT')
-        ) msg = this.$t('pages.claim.noZeroClaim');
-        else if (error.includes('ORACLE_SEVICE_CHECK_CLAIM_FAILED')) msg = this.$t('pages.claim.oracleFailed');
-        else if (error) msg = error;
+        ) {
+          msg = root.$t('pages.claim.noZeroClaim');
+        } else if (error.includes('ORACLE_SEVICE_CHECK_CLAIM_FAILED')) {
```

Comment:
There is and was before a typo here "SEVICE" => "SERVICE". Is this a mistake somewhere on the backend side? If yes I suggest to add comment to prevent someone to fix this. And maybe informing someone that can fix this would be good idea.